src/components/common/form/error-summary/ErrorSummary.spec.jsx

```diff
@@ -4,6 +4,7 @@ import { renderWithRouter } from '../../../../test/helpers/Helpers';
 import ErrorSummary from './ErrorSummary';
 
```

Comment:
Is it worth adding a test for checking the focus?

Reply:
I've added to the already existing test to check that the focus has been set